docs/how-to-guides/renkulab/ssh-into-sessions.rst

```diff
@@ -0,0 +1,255 @@
+        an SSH key pair for you. Once the session is started, Renku creates an
+        SSH connection entry in your local SSH config for that session id. This
+        SSH config entry can be used with your SSH client or tools like VSCode.
+
+
+Open an SSH Session via a Shell
+-------------------------------
+
+You can use :meth:`renku session open <renku.ui.cli.session>` to open an SSH
+connection directly. This will open a terminal in your RenkuLab Project session
+environment.
+
+.. code-block:: console
+
+    $ renku session open --ssh <session-id>
+    venv ▶ ~ ▶ $
+
+(If prompted to accept host keys, confirm with ``yes``).
+
+To exit the SSH shell, simply type ``exit``.
+
+.. note::
+
+    **How do I find my Session ID?** The ``id`` of the session is printed when
+    the session is started by :meth:`renku session start
+    <renku.ui.cli.session>`. In the example above, the session id is
+    ``user-myproject-02a9e407``, so the command to open the SSH session is:
+    ``renku session open --ssh user-myproject-02a9e407``.
+    
+    If you need to find your Session ID again, use
+    :meth:`renku session ls <renku.ui.cli.session>`.
+
+.. note::
+
+    **Can I use normal the** ``ssh`` **command?** Yes! If you prefer, you can
+    use the ``ssh`` command directly rather than the Renku CLI. The ``ssh``
+    command is printed upon starting an ssh session.
+
+        .. code-block:: console
+            :emphasize-lines: 3
+
+            $ renku session start -p renkulab --ssh
+            [...]
+            SSH connection successfully configured, use 'ssh renkulab.io-myproject-02a9e407' to connect.
+
+
+Open an SSH Session in VSCode
+------------------------------
+
+After starting an SSH session, follow these steps to open your session in
+VSCode:
+
+#.  Install the `Remote - SSH Extension <https://marketplace.visualstudio.com/items?itemName=ms-vscode-remote.remote-ssh>`_
+    in VSCode if you don't have it already.
+
+#.  In VSCode, open the "Remote Explorer" in the left bar and make sure "SSH
+    Targets" is selected. Find the SSH connection for the session, e.g.
+    ``renkulab.io-myproject-02a9e407``, and click the "Connect to Host" button.
+
+    .. image:: ../../_static/images/ssh_from_vscode.png
+        :width: 85%
+        :align: center
+        :alt: Opening a RenkuLab SSH Session in VSCode.
+
+#.  Open the "Explorer" in the left bar, and select "Open Folder". Enter
+    ``/home/jovyan/work``.
+
+You can now browse and edit your Renku Project files in VSCode, and run commands
+in the RenkuLab session via the VSCode terminal!
+
+.. image:: ../../_static/images/ssh_session_in_vscode.png
+    :width: 85%
+    :align: center
+    :alt: A RenkuLab SSH Session open in VSCode.
+
+
+For more details on the VSCode SSH extension see
+`the official documentation <https://code.visualstudio.com/docs/remote/ssh>`_.
+
+FAQ
+---
+
+My environment in an SSH session isn't the same as when I'm in a session on RenkuLab - What's going on?
```

Comment:
It took me a while to realize that by "environment" you mean "shell environment" i.e. environment variables. Can we word it maybe in a slightly more obvious way? It's also possible I'm the only one who reads "environment" and thinks "libraries, software, etc"

Reply:
Changed to "shell environment"